src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,118 @@
+package duke;
+
+import java.util.Scanner;
+import java.io.File;
+import java.io.IOException;
+
+
+public class Duke {
+
+    /**
+     * Contains all the driver code currently, will be refactored in the future.
+     *
+     * @param args The array of command line arguments received. Currently unused.
+     */
+    public static void main(String[] args) {
+
+        // Source : https://patorjk.com/software/taag/#p=display&f=Standard&t=Dude
+        String logo2 = " ____            _      \n"
+                + "|  _ \\ _   _  __| | ___ \n"
+                + "| | | | | | |/ _` |/ _ \\\n"
+                + "| |_| | |_| | (_| |  __/\n"
+                + "|____/ \\__,_|\\__,_|\\___|\n";
+
+        System.out.println("Hello from\n" + logo2);
+        System.out.println("Hello! I'm Dude");
+
+        Scanner stdIn = new Scanner(System.in);
+        boolean isRunning = true;
+
+        DataFile dataFile = new DataFile("./duke_data.txt");
+        TaskList taskList = new TaskList(dataFile);
+        System.out.println("What can I do for you?");
+
+        while (isRunning) {
+            String input = getPrompt(stdIn);
+            isRunning = processInput(input, taskList);
+        }
+    }
+
+    static String getPrompt(Scanner sc) {
+        return sc.nextLine();
+    }
+
+    /**
+     * Contains all the input parsing code currently, will be refactored in the future.
+     *
+     * @param str The line of input entered by the user via stdin
+     * @param taskList The instance of Tasklist used to store the current tasks
+     */
```

Comment:
You could add a line for @return to make the method easier to understand for readers.